### PR TITLE
requestParam field value types

### PR DESCRIPTION
**Pull request by tomi-timetac**

### CODE REVIEW

**Comment by tomi-timetac** [comment]
Need to refactor tests as soon as #22 is merged.

**Review by haubitzer — APPROVED** [review]

**Comment by haubitzer** [comment]
@tomi-timetac you can merge this PR before I merge #22

**Comment by tomi-timetac** [comment]
> @tomi-timetac you can merge this PR before I merge #22

Ok, will do. Waiting for @benj56 approve.

**Review by benj56 — APPROVED** [review]

**Pull request merged**
